src/Microsoft.Health.Fhir.TemplateManagement/Models/OrasOperationResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.TemplateManagement.Models
+{
+    public class OrasOperationResult
```

Comment:
Can we make it a general Class like ```OCIOperationResult```? This is returned to customer and seems independent from Oras.

Reply:
Thanks, change it name.